java/arcs/core/data/HandleConnectionSpec.kt

```diff
@@ -7,5 +7,6 @@ import arcs.core.type.Type
 data class HandleConnectionSpec(
     var storageKey: StorageKey?,
     // TODO(shans): type should be a Type, not a Schema
-    val type: Type
+    val type: Type,
+    val ttl: Ttl? = null
```

Comment:
You need to rebase and update ParcelableHandleConnectionSpec + Test

Reply:
i already did